Doc/whatsnew/3.12.rst

```diff
@@ -282,6 +282,10 @@ Removed
   a C implementation of :func:`~hashlib.pbkdf2_hmac()` which is faster.
   (Contributed by Victor Stinner in :gh:`94199`.)
 
+* :mod:`zipimport`: Remove ``find_loader()``, ``find_module()`` and
+  ``load_module()`` methods, deprecated in Python 3.10. Use the ``find_spec()``
```

Comment:
``find_spec()`` only replaces ``find_loader()`` and ``find_module()``, so this sentence is a bit misleading. Same is true for the news entry.